projects/soc-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/entity/ProbationAreaEntity.kt

```diff
@@ -95,6 +97,7 @@ interface ProbationAreaRepository : JpaRepository<ProbationAreaEntity, Long> {
         and pa.selectable = true
         and d.selectable = true
         and b.selectable = true
+        and pa.establishment is null
```

Comment:
Do we need to add the same check to the `probationAreaDistrictsNonSelectable` method?  Also would checking for not equal to Y be better?  I know they're all either Y or null in the DB at the moment, but just in case someone decides to create one with establishment=N.